doc/usermanual/lighttable/panels/tagging.xml

```diff
@@ -29,29 +29,15 @@
             <entry>
               <para>
                 This panel is for managing tags on images. Tags are stored in both sidecar files
-                (.xmp) and within the darktable database for a faster access. The panel is divided
-                into two parts: the upper part contains the tag(s) currently attached to the image
-                under the cursor or the selected image(s) if the mouse is outside the lighttable.
-                The lower part contains a list of all available tags in the database, the dictionary, which can be
-                filtered in the text box above.
+                (.xmp) and within the darktable database for a faster access.
               </para>
               <para>
-                The list of available tags can be displayed in three different manners:
-                <itemizedlist>
-
-                  <listitem><para>
-                    A flat list of tags
-                  </para></listitem>
-
-                  <listitem><para>
-                    A tree view to navigate within your hierarchical tags
-                  </para></listitem>
-
-                  <listitem><para>
-                    A list of suggestions based on already attached tags on selected images.
-                  </para></listitem>
-
-                </itemizedlist>
+                The goal of tags is to add information to the images within a given keyword set.
+                But when the number of tags becomes large it is interesting to organize them in a hierachical manner, as a tree.
```

Comment:
Only a matter of taste: I'd tend to write "You can organize the tags hierarchically as a tree, which is interesting when the number of tags becomes large".
In any case: "hierarchical" (missing an r).

Reply:
Ironically, I worked only on the markup using already present text. But, as I agree that phrasing is sup-optimal, I rephrased towards simplification and (hopefully) clarity.
Frankly speaking, I still do not get the point of using tags as categories (why not two distinct entities?), but that requires an in-depth technical analysis, hence needs to be tackled separately.
I also do not like the fact that lighttable is a view, and tag dictionary is a view, and tree/list view is also.. a view :)